package.json

```diff
@@ -41,6 +42,12 @@
     "minimatch": "^3.0.0",
     "sb-debounce": "^1.0.1"
   },
+  "devDependencies": {
+    "babel-eslint": "^5.0.0",
```

Comment:
`babel-eslint` is being it's notoriously slow self and has yet to release this critical fix.